src/utils/logging.py

```
@@ -0,0 +1,561 @@
+        lines.append(f"**Result:** {score}")
+
+    lines.append("\n### 🧠 Judgment")
+    lines.append(f"```\n{truncate_text(judgment, 800)}\n```")
+
+    if metadata:
+        lines.append(f"\n{format_dict_as_markdown(metadata, '📋 Metadata')}")
+
+    return "\n".join(lines)
+
+
+def format_question_eval_input(
```

Ruff (0.12.2)

`format_question_eval_input` is too complex (13 > 10)

(C901)